Fixed app crash when receiving voip push notification in background

dakenf opened the pull request:
Fixed this issue https://github.com/hoxfon/react-native-twilio-programmable-voice/issues/167
The app is being killed because CallKit was not initialized in time (or not initialized at all for some reason) with JS code. Workaround allows initializing CallKit in obj-c code and handling push notification correctly

fabriziomoscon commented:
Hi @dakenf 
are you sure that `initPushRegistry` will be called once?

Maybe this should be removed:
https://github.com/hoxfon/react-native-twilio-programmable-voice/blob/master/ios/RNTwilioVoice/RNTwilioVoice.m#L71
?

dakenf commented:
No, it is being called twice if you add the code from the workaround. It does not seem to cause any side effects (at least we haven't seen any) but i will make changes to check if PKPushRegistry was already created. Will do some testing and update the PR after the Christmas

jdegger commented:
Hi @dakenf do you have an update for us already :)?

fabriziomoscon commented:
@dakenf is your intention to add any new commit to this?
Anyhow, could I ask to apply this fix to #164 ?

dakenf commented:
@fabriziomoscon @jdegger Hello everyone, sorry for very late reply. Was being busy on other projects. I've updated Twilio SDK to the latest 6.3.0 version, fixed issues with PushKit initialzation that caused crash if code did not respond to VoIP push in time and added info about that to Readme.md

Also added a new event for iOS that is triggered when user taps on the call in system call log. It opens the app and sends phone number (or client ID) to js
